### PR TITLE
Map/NoName - Building Smoothing

### PR DESCRIPTION
**Pull request by m4rtint**
- Added Prefabs for buildings
- Smoothed out buildings
- Changed lighting for no name

image

### CODE REVIEW

**Comment by mikeslgoh** [comment]
[Smooth edges investigation (1.0)](https://trello.com/c/feMZCAmx/709-smooth-edges-investigation-10)

File: ProjectSettings/QualitySettings.asset [hunk 1]
```diff
@@ -201,7 +201,7 @@ QualitySettings:
     skinWeights: 4
     textureQuality: 0
     anisotropicTextures: 0
-    antiAliasing: 0
+    antiAliasing: 4
```
Comment (mikeslgoh) [inline ProjectSettings/QualitySettings.asset:204]: Is there a reason why this changed?

Comment (m4rtint) [reply ProjectSettings/QualitySettings.asset:204]: This keeps popping up, not sure why. I'll probably go through it more times and it'll be changed several times back to 4.

**Comment by m4rtint** [comment]
[Clean up No Name Island](https://trello.com/c/vVFN01Je/723-clean-up-no-name-island)

**Pull request merged**